### DISCUSSION TITLE
supabase onboarding

### DISCUSSION THREAD

**Discussion by PSV-07** [supabase/supabase#29775] [Contribute to Supabase]
hello guys i 'm a new user of supabase in the process of inboarding i ran into a problem it wants my organisation name but doesn't matter what name i type into it just reloads the same shitty page for the record i'm from india and help could be appreciated

**Comment by J0** [comment]
Hey @PSV-07,

Thanks for flagging the issue and sorry for the inconvenience - could you open a ticket at supabase.help ? We're looking into the issue.

Thanks

**Reply by PSV-07** [reply]
Hey genius it's again asking me for an active organisation I mean how can I name an organisation that hasn't been. Created yet I face that glitch which keeps loading the organisation name page when I try to create one I need immediate assistance it's an emergency please look up into the issue

**Reply by kevcodez** [reply]
Just a reminder to keep it classy. We'll look into adding the ability to create support tickets without any organizations. Note that you can always just email support@supabase.io to reach the team directly.

**Comment by GaryAustin1** [comment]
https://status.supabase.com/incidents/f6rzxqqvxcpk

**Comment by kevcodez** [comment]
This has been resolved, reloading the page once should do the trick.
